packages/libs/wallet-sdk/etc/wallet-sdk.api.md

```diff
@@ -22,8 +22,12 @@ export type ChainwebHostGenerator = (options: {
 // @public (undocumented)
 export type GraphqlHostGenerator = (options: {
     networkId: string;
+    graphType?: GraphType;
 }) => string;
 
+// @public (undocumented)
+export type GraphType = 'kadena' | 'hackachain';
+
```

Comment:
Why do we distinguish between the two graphs?
It should be that everything works on both endpoints, EXCEPT the accountsByPublicKey. All the others should be backwards compatible (potentially with deprecation message, but not failing)